text/char-rnn/char-rnn.jl

```diff
@@ -51,10 +50,9 @@ function train(; kws...)
 
     # Constructing Model
     m = Construct_model(N)
-    m = gpu(m)
 
     function loss(xs, ys)
-      l = sum(crossentropy.(m.(gpu.(xs)), gpu.(ys)))
+      l = sum(logitcrossentropy.(m.(xs), ys))
```

Comment:
is it ok to have a broadcast here?

Reply:
Yes, here without the broadcast, it throws the following error:
```
MethodError: no method matching isless(::Array{Float32,2}, ::Array{Float32,2})
Closest candidates are:
  isless(!Matched::Missing, ::Any) at missing.jl:87
  isless(::Any, !Matched::Missing) at missing.jl:88
```
as here `xs` is an array of 50 elements where each element is a 2D array (N x 50) which is input for the model. However I think here it would be better to have `mean` rather than `sum`